### DISCUSSION TITLE
Apply Hotfix from #229

### DISCUSSION THREAD

**Discussion by theAkito** [lencx/Noi#231] [Q&A]
https://github.com/lencx/Noi/issues/229#issuecomment-2340158217

@mitch133

I recommend discussing your individual issue here, so the original issue does not get bloated.

Could you post screenshots of what you changed and how it looks and in what folders the changes took place?

**Comment by mitch133** [comment]
Ok here you have a video of what I observe on my 2 PC's, even after applying the "patch" (3 files replaced)

https://github.com/user-attachments/assets/ee6d98e2-dd0d-456c-811e-3c861c08f8d3

**Comment by mitch133** [comment]
You can see on the video that there's a specific problem with ChatGPT
Not at the beginning within NOI Ask (as I mentionned as a workaround on the other thread) but whenever you select ChatGPT on the side menu, it happens, you cannot select other GPTs anymore or even move to other AIs
You have to come back to NOI Ask to unlock everything

**Reply by theAkito** [reply]
> You can see on the video that there's a specific problem with ChatGPT Not at the beginning within NOI Ask (as I mentionned as a workaround on the other thread) but whenever you select ChatGPT on the side menu, it happens, you cannot select other GPTs anymore or even move to other AIs You have to come back to NOI Ask to unlock everything

Could you test a workflow, which is *only* using ChatGPT and nothing else?

I tested it with ChatGPT and a single other AI. But not the ones you used and it works.

Try using only ChatGPT after opening Noi. Click around only in ChatGPT. In the video you already showed, that the patch worked enough to not be locked out from the beginning, at least.

**Comment by mitch133** [comment]
I tested it (and actually I've been observing this for a while :)), it doesn't work.

And don't want to disappoint you, but not being locked out anymore from the beginning is not related to the patch. 
Actually I still am after the patch.
As I told you (and as you can see in the video), what unlocks the access to other AIs is the fact that I leave the "start screen" of NOI. In my case I click on NOI Ask, but I guess it can work clicking somewhere else.

But the problem always appears when you click on ChatGPT (or start NOI with ChatGPT)

https://github.com/user-attachments/assets/efdfa357-efb2-46e5-a41a-d1978f5ffd9a

**Reply by theAkito** [reply]
I tested everything, including Noi Ask, without Noi Ask, just ChatGPT, other AI providers, etc. Everything works fine!

**Reply by mitch133** [reply]
ok, what's your advice then?

**Comment by theAkito** [comment]
@mitch133 

It works on Windows 11.

https://github.com/lencx/Noi/issues/229#issuecomment-2365451087
